vision/detect/generated/sample-template.go

```diff
@@ -306,3 +306,60 @@ func detectLogos(w io.Writer, file string) error {
 
 	return nil
 }
+
+// [START vision_detect_async_document{REGION_TAG_PARAMETER}]
+
+// TODO: write docs
+// detectAsyncDocument
+func detectAsyncDocument(w io.Writer, gcs_source_uri, gcs_destination_uri string) error {
```

Comment:
Please use camelCase for variables in Go. See https://golang.org/doc/effective_go.html#mixed-caps.